package-res/resources/web/pentaho/lang/EventSource.js

```diff
@@ -325,10 +326,11 @@ define([
         result = this._emit(event);
```

Comment:
We must discuss this with Marco, because I had the impression we wanted to interrupt the whole pipeline in the "will" case. In the "did" and "reject" cases, it might make sense to have a try-catch per listener.